### PR TITLE
Added nino, gender and previous names to person view

### PR DESCRIPTION
**Pull request by hortha**
### Context

Users of the support desk need gender and NINO to help identify the correct record.

### Changes proposed in this pull request

Add NINO, Gender and previous name(s) to the summary list on the person view, as per the designs.

### Guidance to review

CRM query, UI + tests

### Checklist

-   [x] Attach to Trello card
-   [ ] Rebased master
-   [ ] Cleaned commit history
-   [x] Tested by running locally

### CODE REVIEW

**Comment by hortha** [comment]
[Add NINO and Gender to person view](https://trello.com/c/c1sIxJ8p/1232-add-nino-and-gender-to-person-view)

File: TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/TagHelpers/UseEmptyFallbackTagHelper.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+using System.Text.Encodings.Web;
+using Microsoft.AspNetCore.Mvc.TagHelpers;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace TeachingRecordSystem.SupportUi.TagHelpers;
+
+[HtmlTargetElement("span", Attributes = "use-empty-fallback")]
```
Comment (gunndabad) [inline TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/TagHelpers/UseEmptyFallbackTagHelper.cs:7]:
```suggestion
[HtmlTargetElement("*", Attributes = "use-empty-fallback")]
```

File: TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/Index.cshtml [hunk 2]
```diff
@@ -7,32 +7,61 @@
...
+}
+
 <govuk-summary-card>
     <govuk-summary-card-title>Personal Details</govuk-summary-card-title>
     <govuk-summary-list data-testid="personal-details">
         <govuk-summary-list-row>
             <govuk-summary-list-row-key>Name</govuk-summary-list-row-key>
             <govuk-summary-list-row-value data-testid="personal-details-name">@Model.Person.FullName</govuk-summary-list-row-value>
         </govuk-summary-list-row>
+        @if (Model.Person.PreviousNames is not null && Model.Person.PreviousNames.Length > 0)
```
Comment (gunndabad) [inline TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/Index.cshtml:32]: Do we need the null check?

File: TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/TagHelpers/UseEmptyFallbackTagHelper.cs (hunk 1 above)
Comment (gunndabad) [reply TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/TagHelpers/UseEmptyFallbackTagHelper.cs:7]: We could avoid adding extra `span`s if we can use this everywhere (thought it probably won't work on the `govuk-` elements unless we lower the `Order`)

Comment (hortha) [reply TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/TagHelpers/UseEmptyFallbackTagHelper.cs:7]: it didn't work when I tried with govuk-summary-list-row-value as it seemed to ignore any changes made to content when it came to rendering

**Pull request closed without merging**